src/kbmod/run_search.py

```diff
@@ -332,40 +334,50 @@ def _count_known_matches(self, result_list, img_info, search):
         search : `kbmod.search.stack_search`
             A stack_search object containing information about the search.
         """
-        # Lookup the known objects using either SkyBoT or the JPL API.
+        # Get the image metadata
+        im_filepath = self.config["im_filepath"]
+        filenames = sorted(os.listdir(im_filepath))
+        image_list = [os.path.join(im_filepath, im_name) for im_name in filenames]
+        metadata = koffi.ImageMetadataStack(image_list)
+        
+        # Get the pixel positions of results
+        ps_list = []
+        
+        for row in result_list.results:
+            pix_pos_objs = search.get_mult_traj_pos(row.trajectory)
+            pixel_positions = list(map(lambda p : [p.x, p.y], pix_pos_objs))
+            ps = koffi.PotentialSource()
+            ps.build_from_images_and_xy_positions(pixel_positions, metadata)
+            ps_list.append(ps)
+        
         print("-----------------")
-        known_objects = KnownObjects()
+        matches = {}
+        known_obj_thresh = self.config["known_obj_thresh"]
+        min_obs = self.config["known_obj_obs"]
         if self.config["known_obj_jpl"]:
             print("Quering known objects from JPL")
-            known_objects.jpl_query_known_objects_mult(img_info)
+            matches = koffi.jpl_query_known_objects_stack(
+                ps_list, metadata, min_observations=min_obs, tolerance=known_obj_thresh
```

Comment:
This is a personal preference but I always like when keyword name is included and each argument is on its own line.

```
koffi.jpl_query_known_object_stack(
    potential_sources=ps_list,
    image=metadata,
    min_observations=min_obs,
    tolerance=known_obj_tresh
)
```

Reply:
seems like a good idea to do actually! should be consistent across all params anyway